File: Software Development Module/Week2/AreaCircle.java
/*
AreaCircle.java
Billy Cussen
07/07/2020
*/

//This class will ask the user for the Radius of a Circle and produce its Area


import java.util.Scanner;

public class AreaCircle {
    public static void main(final String[] args) {

        Scanner keyboard = new Scanner(System.in);

        System.out.print("Please enter the radius of the Circle: ");
        double radius = keyboard.nextDouble();
        keyboard = null;
        
        double area = Math.PI * Math.pow(radius, 2);

        System.out.println("Area of the Circle is: "+area);


    }
}

File: Python/Software Development Module - Python/Week2/AreaCircle.py
"""
AreaCircle.py
Billy Cussen
23/09/2020
"""
from math import pi

inputCheck = False

while inputCheck == False:

    try:
        radius = float(input("Please enter the Circles Radius: "))
    except:
        print("You need to enter a Number!")
    else:
        if radius <= 0:
            print("Your input needs to be 1 or greater!")
        else:
            inputCheck = True
            print("The Area of this circle is: "+str(round(pi*(radius**2),2)))

print ("Thanks for using my App!")